common/changes/@gooddata/sdk-ui-all/RAIL-2317_2020-10-20-10-06.json

```diff
@@ -2,7 +2,7 @@
     "changes": [
         {
             "packageName": "@gooddata/sdk-ui-all",
-            "comment": "Added hooks alternative for Execute component - useCancelablePromise, useDataView, useExecution and useDataExport",
+            "comment": "Alternatives to hooks (useCancelablePromise, useDataView, useExecution, and useDataExport) are added to the Execute component.",
```

Comment:
This is inverted: it should be something like
> Hook alternatives for Execute component are added.

In other words, we did not have hooks and now we do.

Reply:
Gotcha, thanks!